internal/store/engine_state.go

```diff
@@ -85,15 +85,7 @@ type EngineState struct {
 	TokenKnownUnregistered                      bool // to distinguish whether an empty TiltCloudUsername means "we haven't checked" or "we checked and the token isn't registered"
 	WaitingForTiltCloudUsernamePostRegistration bool
 
-	DockerPruneSettings DockerPruneSettings
-}
-
-type DockerPruneSettings struct {
-	Disabled bool
-}
-
-func defaultDockerPruneSettings() DockerPruneSettings {
-	return DockerPruneSettings{Disabled: true} // for now, always disabled until we can configure it
+	DockerPruneSettings model.DockerPruneSettings
```

Comment:
This looks weird to me. What's happening here? Is this returning the empty value of `model.DockerPruneSettings`?

Reply:
Yep: if we don't bubble the TiltfileLoadResults all the way through the engine state, then we get the empty value of `DockerPruneSettings` (where `Enabled = false` -- so in case like `tilt down`, we're not running the pruner at all).

Reply:
Can we be explicit about what the default values are rather than returning empty values for everything?